structs/student_validator.c

```diff
@@ -0,0 +1,265 @@
+#include <stdio.h>
```

Comment:
Either remove this file (it is duplicating the `arrays/student_validator_using_pointers.c`) or make it correct